src/rx.cpp

```diff
@@ -661,15 +682,14 @@ void Aggregator::process_packet(const uint8_t *buf, size_t size, uint8_t wlan_id
 
             init_fec(new_session_data->k, new_session_data->n);
 
-            fprintf(stdout, "%" PRIu64 "\tSESSION\t%" PRIu64 ":%u:%d:%d\n", get_time_ms(), epoch, WFB_FEC_VDM_RS, fec_k, fec_n);
```

Comment:
@vertexodessa Why removing these two lines?

Reply:
they are replaced with 
`WFB_LOG(LogLevel::ALWAYS`
which unconditionally logs without prefix.
Is it fine to log these line to stderr?

Reply:
No. This is machine-readable telemetry counters and that is why it passed to stdout (also there is flush on the next line).
Machine readable logs are passed to stdout
Humane readable logs to stderr

Reply:
@vertexodessa Please rollback removing of this fprintf and fflush